metaflow/metaflow_config.py

```diff
@@ -242,6 +242,7 @@
 KUBERNETES_SERVICE_ACCOUNT = from_conf("KUBERNETES_SERVICE_ACCOUNT")
 # Default node selectors to use by K8S jobs created by Metaflow - foo=bar,baz=bab
 KUBERNETES_NODE_SELECTOR = from_conf("KUBERNETES_NODE_SELECTOR", "")
+KUBERNETES_TOLERATIONS = from_conf("METAFLOW_KUBERNETES_TOLERATIONS")
```

Comment:
This needs a default value of `""` similar to `KUBERNETES_NODE_SELECTOR` and `KUBERNETES_SECRETS`.